app/views/settings_dialog.py

```diff
@@ -686,6 +688,50 @@ def _do_todds_tab(self) -> None:
         )
         group_layout.addWidget(self.todds_overwrite_checkbox)
 
+    def _do_themes_tab(self) -> None:
+        tab = QWidget()
+        self.tab_widget.addTab(tab, "Themes")
+
+        tab_layout = QVBoxLayout(tab)
+        tab_layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+
+        group_box = QGroupBox()
+        tab_layout.addWidget(group_box)
+
+        group_layout = QVBoxLayout()
+        group_box.setLayout(group_layout)
+
+        self.themes_label = QLabel("Themes Support")
```

Comment:
Is this label necessary, there is only one section and the tab title of Themes should be enough as a descriptor? Additionally, if chosen to be kept, support should be lowercase for consistency.

Reply:
removed Done